README.md

```diff
@@ -207,6 +207,12 @@ Example:
 - Twitter: [@twitterhandle](https://twitter.com/@rksne123)
 - LinkedIn: [LinkedIn](www.linkedin.com/in/mariano-benjamin-mendez-garcia-a50856177)
 
+👤 **Maedea9**
+
+- GitHub: [@githubhandle](https://github.com/maedea)
```

Comment:
- [ ] [OPTIONAL] I would like to strongly recommend you customize your license by adding you names and application names in order to make it your own and your readme more professional 👍